beacon_node/execution_layer/src/lib.rs

```diff
@@ -235,7 +235,7 @@ impl<T: EthSpec> ExecutionLayer<T> {
     }
 
     /// Note: this function returns a mutex guard, be careful to avoid deadlocks.
-    async fn execution_blocks(
+    pub async fn execution_blocks(
```

Comment:
```suggestion
    async fn execution_blocks(
```

This doesn't need to be public

Reply:
Oops sorry, Was using this for some testing.